public_html/wp-content/mu-plugins/blocks/assets/src/organizers/edit.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { isUndefined, pickBy, split } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+const apiFetch                = wp.apiFetch;
+const { withSelect }          = wp.data;
+const { Component, Fragment } = wp.element;
+const { addQueryArgs }        = wp.url;
+
+/**
+ * Internal dependencies
+ */
+import OrganizersBlockControls     from './block-controls';
+import OrganizersInspectorControls from './inspector-controls';
+import OrganizersToolbar           from './toolbar';
+
+export const ORGANIZERS_ICON = 'groups';
+
+const blockData = window.WordCampBlocks.organizers || {};
+const MAX_POSTS = 100;
+
+const ALL_POSTS_QUERY = {
+	orderby  : 'title',
+	order    : 'asc',
+	per_page : MAX_POSTS,
+	_embed   : true,
+};
+
+const ALL_TERMS_QUERY = {
+	orderby  : 'name',
+	order    : 'asc',
+	per_page : MAX_POSTS,
+};
+
+class OrganizersEdit extends Component {
+	constructor( props ) {
+		super();
+
+		this.state = {
+			allSpeakerPosts : null,
+			allSpeakerTerms : null,
+		};
+	}
+
+	componentWillMount() {
+		this.isStillMounted = true;
+
+		const allSpeakerPosts = apiFetch( {
+			path: addQueryArgs( `/wp/v2/organizers`, ALL_POSTS_QUERY ),
+		} );
+		const allSpeakerTerms = apiFetch( {
+			path: addQueryArgs( `/wp/v2/organizer_team`, ALL_TERMS_QUERY ),
+		} );
```

Comment:
`[MIGHT]` I'm getting an `Unhandled promise rejection` error here. It's probably just because I haven't finished setting up the API endpoint the way that the block expects, but the underlying issue is that it feels like a code smell that we're not catching rejections.

https://stackoverflow.com/questions/40500490/what-is-an-unhandled-promise-rejection

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

It seems equivalent to doing a `try` without a `catch` in PHP. Should we add `catch` everywhere we use promises?

@coreymckrill , @vedanshujain

Reply:
In the Sessions block I added `catch` in the `SessionsSelect` component, because that's where the promises ultimately get resolved.

Reply:
@iandunn you will probably get this when request fails, I am not sure if we can do anything inside `catch` statement which will fix it and is not complicated. So user would have to reload their page in case this happens. 
Please feel free too add a new issue for our standard behavior in case network call fails (maybe display a notification on top of editor?)

Reply:
Good idea, moved to https://github.com/WordPress/wordcamp.org/issues/46